swarm/api/api.go

```diff
@@ -204,52 +200,55 @@ func (m *MultiResolver) getResolveValidator(name string) ([]ResolveValidator, er
 	return rs, nil
 }
 
+// SetNameHash - on a MultiResolver
 func (m *MultiResolver) SetNameHash(nameHash func(string) common.Hash) {
 	m.nameHash = nameHash
 }
 
 /*
-Api implements webserver/file system related content storage and retrieval
+API implements webserver/file system related content storage and retrieval
 on top of the dpa
 it is the public interface of the dpa which is included in the ethereum stack
 */
-type Api struct {
+type API struct {
 	resource *mru.Handler
 	dpa      *storage.DPA
 	dns      Resolver
 }
 
-//the api constructor initialises
-func NewApi(dpa *storage.DPA, dns Resolver, resourceHandler *mru.Handler) (self *Api) {
-	self = &Api{
+// NewAPI - the api constructor initialises
+func NewAPI(dpa *storage.DPA, dns Resolver, resourceHandler *mru.Handler) (self *API) {
+	self = &API{
 		dpa:      dpa,
 		dns:      dns,
 		resource: resourceHandler,
 	}
 	return
 }
 
-// to be used only in TEST
-func (self *Api) Upload(uploadDir, index string, toEncrypt bool) (hash string, err error) {
-	fs := NewFileSystem(self)
+// Upload - to be used only in TEST
+func (a *API) Upload(uploadDir, index string, toEncrypt bool) (hash string, err error) {
+	fs := NewFileSystem(a)
 	hash, err = fs.Upload(uploadDir, index, toEncrypt)
 	return hash, err
 }
 
-// DPA reader API
-func (self *Api) Retrieve(addr storage.Address) (reader storage.LazySectionReader, isEncrypted bool) {
-	return self.dpa.Retrieve(addr)
+// Retrieve - DPA reader API
+func (a *API) Retrieve(addr storage.Address) (reader storage.LazySectionReader, isEncrypted bool) {
+	return a.dpa.Retrieve(addr)
 }
 
-func (self *Api) Store(data io.Reader, size int64, toEncrypt bool) (addr storage.Address, wait func(), err error) {
+// Store DPA store API
+func (a *API) Store(data io.Reader, size int64, toEncrypt bool) (addr storage.Address, wait func(), err error) {
 	log.Debug("api.store", "size", size)
-	return self.dpa.Store(data, size, toEncrypt)
+	return a.dpa.Store(data, size, toEncrypt)
 }
 
+// ErrResolve declaration
 type ErrResolve error
 
-// DNS Resolver
-func (self *Api) Resolve(uri *URI) (storage.Address, error) {
+// Resolve - DNS Resolver
```

Comment:
Resolve resolves a uri to an Address using the multiresolver

Reply:
URI should be capitalized

Reply:
```
Resolve resolves a URI to an Address using the MultiResolver.
```